modules/docs/arrow-docs/docs/_includes/_home-doc-versions.html

```diff
@@ -0,0 +1,32 @@
+<div class="masthead-doc-version">
+  <ul>
+    <li>
+      <a class="masthead-button secondary-button button"
+        href="{{ previous_version.url }}">
+        <span>{{ previous_version.title }}</span>
+        <span class="secondary-doc-version-text">
+          {{ site.data.doc-versions.previous }}
+        </span>
+      </a>
+    </li>
+
+    <li>
+      <a class="masthead-button button" href="{{ stable_version.url }}">
+        <span>{{ stable_version.title }}</span>
+        <span class="secondary-doc-version-text">
+          {{ site.data.doc-versions.stable }}
+        </span>
+      </a>
+    </li>
+
+    <li>
+      <a class="masthead-button secondary-button button"
+        href="{{ next_version.url }}">
```

Comment:
Is this not pointing to the `docs` path deliberately?  As the previous CTA button behaviour was that way, I find it a bit weird that they don't lead me to the docs, but maybe it is what we want.